addons/l10n_ro_efactura/models/account_move.py

```diff
@@ -250,13 +250,14 @@ def _l10n_ro_edi_fetch_invoice_sending_documents(self):
                     company=invoice.company_id,
                     key_download=result['key_download'],
                     session=session,
+                    status=result['state_status'],
                 )
                 to_delete_documents |= invoice._l10n_ro_edi_get_sending_and_failed_documents()
                 final_result['key_loading'] = active_sending_document.key_loading
-                if 'error' in final_result:
+                if final_result['error']:
```

Comment:
mmh... you sure? because from the  previous code, it seems obvious the key 'error' could be _not_ present in final_result, whereas here you assume it'll always be :suspect: 

ok, turns out that with your code you'll always return the key error... but why not use defensive code? 
```suggestion
                if final_result.get('error'):
```
that's nitpicking, feel free to ignore, but that code made me raise my eyebrown before I checked on `_request_ciusro_download_answer`... so you know